src/infrastructure/MosaicService.js

```diff
@@ -123,7 +123,13 @@ class MosaicService {
    		data: mosaicInfos.data.map((mosaic) => ({
    			...mosaic,
    			ownerAddress: mosaic.address,
-   			mosaicAliasNames: this.extractMosaicNamespace(mosaic, mosaicNames)
+   			mosaicAliasNames: this.extractMosaicNamespace(mosaic, mosaicNames),
+   			mosaicFlags: {
+   				mutable: mosaic.supplyMutable,
```

Comment:
Should we keep property name `supplyMutable` consistent?

```suggestion
   				supplyMutable: mosaic.supplyMutable,
```